docs/guides/rendering.md

```diff
@@ -1,445 +1,350 @@
+An instance of one of these callables captures all the state needed to perform slicing,
+so that it can be executed asynchronously on another thread without needing to guard competing access to that state with locks.
+
+Unfortunately, these new additions make following the old synchronous slicing code paths more complicated.
+But eventually we hope to mostly remove those complications and make both synchronous and asynchronous slicing consistent and easy enough to follow.
+
+#### Future
+
+The current experimental asynchronous slicing approach is limited.
+While it prevents napari from blocking the main thread, fetching and rendering the data in view can still be slow.
+
+Most large image viewers improve on this experience with by progressively fetching and rendering chunks or tiles of data.
+This allows some data to be presented quickly rather than waiting for everything in view,
+which often results in a much better user experience when fetching the data is slow.
+
+These efforts across all layers are generally tracked by [issue #5942](https://github.com/napari/napari/issues/5942).
+Currently, the most focus is on the image layer in [issue #5561](https://github.com/napari/napari/issues/5561),
+with lots of progress towards that in [PR #6043](https://github.com/napari/napari/pull/6043).
+
+## Drawing
+
+After the current view of a layer's data has been sliced into RAM, the data is pushed to VRAM with any associated transforms and parameters.
+
+Each napari layer type has a corresponding vispy layer type.
+For example, the [`VispyImageLayer`](https://github.com/napari/napari/blob/5e8dc098cb213c5f963524e619f223ad4fe90be8/napari/_vispy/layers/base.py#L21)
+corresponds to the {class}`Image<napari.layers.Image>` layer.
+Each instance of a layer also has a corresponding instance of its vispy layer.
+These correspondences can be found [`VispyCanvas.layer_to_visual`](https://github.com/napari/napari/blob/5e8dc098cb213c5f963524e619f223ad4fe90be8/napari/_vispy/canvas.py#L69).
+
+The vispy layer instance has a reference to its corresponding layer.
+Updates to the layer's state and its current slice trigger the vispy layer are handled using [napari's event system](connect-napari-event).
+Of particular interest here is the [`Layer.events.set_data` event](layer-events), which is connected to the abstract method
+[`VispyBaseLayer._on_data_change`](https://github.com/napari/napari/blob/5e8dc098cb213c5f963524e619f223ad4fe90be8/napari/_vispy/layers/base.py#L74).
+This event is triggered when slicing is finished and the latest slice state can be read.
+
+Each vispy layer type is responsible for implementing the `_on_data_change` method.
+This implementation of this method should read the updated state from the layer, then update the vispy layer appropriately.
+In turn, vispy handles make the appropriate updates to VRAM and executes any programs needed to update the display on napari's canvas.
```

Comment:
```suggestion
In turn, vispy handles making the appropriate updates to VRAM and executes any programs needed to update the display on napari's canvas.
```

Reply:
Using `makes` instead of `handles making` instead.